src/Microsoft.Management.Configuration/Microsoft.Management.Configuration.idl

```diff
@@ -766,6 +774,12 @@ namespace Microsoft.Management.Configuration
 
             // The execution policy to apply; must be set before taking actions with the processor.
             PwshConfigurationProcessorPolicy Policy;
+
+            // The location to install modules. Default is CurrentUser

```

Comment:
I think we should change the default to `WinGetModulePath` to avoid the problem by default rather than requiring opt-in.